fud/fud/json_to_dat.py

```diff
@@ -38,6 +240,7 @@ def convert2dat(output_dir, data, extension):
 
 
 # converts a directory of *.dat files back into a json file
+# TODO: Figure out extention for this
```

Comment:
is this still relevant?

Reply:
I don't think so. I think you can get rid of it